TRACONBoundaries.geojson

```diff
@@ -503,6 +504,7 @@
     {"type":"Feature","properties":{"id":"UMMS","prefix":["UMMS"],"name":"Minsk Approach"},"geometry":{"type":"MultiPolygon","coordinates":[[[[26.98565028,54.32017167],[26.79305583,54.03444333],[26.72250000,53.89472167],[26.72249944,53.89472167],[26.82305583,53.60138889],[26.95638861,53.45638833],[27.24833278,53.21638861],[27.67638861,53.11805583],[28.13027722,53.11194333],[28.37416667,53.10777722],[28.67694389,53.10194389],[28.75194361,53.14861083],[28.88166667,53.22888889],[29.07972167,53.35000003],[29.17027750,53.43722194],[29.22916667,53.49111056],[29.27333278,53.58166639],[29.38972194,53.81833222],[29.33305556,53.97805583],[29.30138861,54.03499972],[29.19500000,54.13277750],[29.01527778,54.29694389],[28.92444361,54.37888833],[28.80611083,54.48583278],[28.52138889,54.51805583],[28.10972194,54.58305528],[27.69472194,54.58805583],[27.44138861,54.55138833],[27.02027722,54.37138833]]]]}},
     {"type":"Feature","properties":{"id":"UMMS","prefix":["UMMS_R"],"name":"Minsk Radar"},"geometry":{"type":"MultiPolygon","coordinates":[[[[28.75027778,53.52250000],[28.64277750,53.45305556],[28.47083333,53.39444444],[28.27638861,53.37416667],[28.08194444,53.39388833],[27.89416667,53.45972222],[27.52916639,53.39222222],[27.14305556,53.97638889],[27.22694444,54.15250000],[27.32194417,54.25722222],[27.46777750,54.33749944],[27.65416639,54.38305556],[27.85583306,54.38888861],[28.04888889,54.35388889],[28.22333306,54.27388889],[28.71333333,53.94138861],[28.81694444,53.83694417],[28.85277778,53.72388861],[28.81500000,53.59583306]]]]}},
     {"type":"Feature","properties":{"id":"UMOO","prefix":["UMOO_TWR"],"name":"Homiel Tower"},"geometry":{"type":"MultiPolygon","coordinates":[[[[30.61805556,54.17000000],[30.83999972,53.86416667],[30.75805583,53.64500000],[30.43305583,53.55166667],[30.14972167,53.49166694],[29.76055583,53.61694389],[29.38972194,53.81833222],[29.33305556,53.97805583],[29.44138861,54.18333278],[29.50777778,54.25416667],[29.99305556,54.21861083],[30.23000000,54.16111028]]]]}},
+    {"type":"Feature","properties":{"id":"URKA","prefix":["URKA_R"],"name":"Anapa Krug"},"geometry":{"type":"MultiPolygon","coordinates":[[[[37.583330000000004,46.1],[37.0775,46.1],[37.0775,46.1],[37.01667,46],[37.01667,46],[36.68333,45.4],[36.68333,45.4],[36.53333,45.2],[36.53333,45.2],[36.6,44.96667],[36.6,44.96667],[36.666669999999996,44.51667],[36.666669999999996,44.51667],[36.64,44.4],[36.64,44.4],[37.25,44.4],[37.25,44.4],[37.40361,44.47972],[37.40361,44.47972],[37.56167,44.63139],[37.56167,44.63139],[37.87833,45.01139],[37.87833,45.01139],[37.91833,45.06667],[37.91833,45.06667],[37.95,45.36667],[37.95,45.36667],[37.583330000000004,45.55],[37.583330000000004,45.55],[37.583330000000004,46.1]]]]}},
```

Comment:
No huge deals here, but there are a few border overlaps here. 

These two are in the North-East part:
![Screen Shot 2022-10-07 at 12 14 40](https://user-images.githubusercontent.com/967239/194530931-88572018-ff18-4bf4-a22d-c9155c66c9d6.png)
![Screen Shot 2022-10-07 at 12 15 15](https://user-images.githubusercontent.com/967239/194530948-8fde2c4b-ea1b-4e6e-b223-30c83dfa5acd.png)

This is in the South-East:

![Screen Shot 2022-10-07 at 12 15 46](https://user-images.githubusercontent.com/967239/194531047-9a07c7b8-fcff-4686-9024-309ad23afc31.png)
![Screen Shot 2022-10-07 at 12 15 58](https://user-images.githubusercontent.com/967239/194531082-99a94038-68da-4a63-b837-5840bfb37be6.png)

Reply:
Checked. Will fix